build.gradle

```diff
@@ -19,6 +19,10 @@ buildscript {
                 }
             }
             repositoryHandler.mavenCentral()
+            repositoryHandler.maven {
+                url "http://3.130.234.198:8081/nexus/content/repositories/trident/"
+                content { includeGroup "org.tron.trident" }
+            }
```

Comment:
is this ok & safe to have it like this? I guess so (if we check the checksums)
 the "includeGroup" should mean this repo would be used only for the given group

I wanted to use jitpack that we have already instead but it cannot compile their github repo with java 8 (we would have to fork it and tell jitpack how to compile it)